modules/ui/sections/preset_fields.js

```diff
@@ -64,8 +64,17 @@ export function uiSectionPresetFields(context) {
 
             _fieldsArr = [];
 
+            // Ideally, everything in OpenHistoricalMap is dated and sourced.
+            let coreKeys = ['start_date', 'end_date', 'source'];
```

Comment:
I didn’t make the License field visible by default. I figure that putting the License field even above the Name field would unduly pressure some mappers into setting a license other than the default CC0, which is allowed but not ideal: OpenHistoricalMap/issues#367 and OpenHistoricalMap/issues#377. By contrast, dating and sourcing everything is simply good practice in historical geography.